gef-ui/templates/partials/application-activity.njk

```diff
@@ -0,0 +1,31 @@
+{%- from "moj/components/timeline/macro.njk" import mojTimeline -%}
+
+{% extends "index.njk" %}
+
+{% block pageTitle %}
+  Application Activities
+{% endblock %}
+
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{{ govukPhaseBanner({
+  tag: {
+    text: "beta"
+  },
+  html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+}) }}
```

Comment:
Wondering if the phase banner should be a seperate ticket; looks like it will only be rendered for this page?